lib/tls.js

```diff
@@ -22,9 +22,9 @@ exports.DEFAULT_CIPHERS =
 
 exports.DEFAULT_ECDH_CURVE = 'prime256v1';
 
-exports.getCiphers = internalUtil.cachedResult(() => {
-  return internalUtil.filterDuplicateStrings(binding.getSSLCiphers(), true);
-});
+exports.getCiphers = internalUtil.cachedResult(
```

Comment:
Unrelated change. I don't object to cleaning up `lib/`'s use of rocket functions, but I think they should be standalone commits, ideally a standalone commit per file that makes every use in the file consistent (this might or might not be the only such usage, I can't tell from the diff, and the commit message doesn't even mention that there is any cleanup occuring) or just to the entire lib directory to clean up a specific usage aspect.

I also don't think commits called `test: ...` should touch lib, unless the lib change really clearly is something needed to support the test code (an example might be exporting an internal so that it can be directly used in a unit test)